pom.xml

```diff
@@ -731,15 +731,14 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-javadoc-plugin</artifactId>
-					<version>3.3.1</version>
+					<version>3.4.1</version>
 					<configuration>
 						<encoding>utf8</encoding>
 						<source>11</source>
 						<doclint>none</doclint>
 						<additionalOptions>
 							<additionalOption>-html5</additionalOption>
 						</additionalOptions>
-						<additionalJOption>${additional.j.option}</additionalJOption>
 						<additionalDependencies>
 							<additionalDependency>
 								<groupId>org.elasticsearch</groupId>
```

Comment:
Also updated this one. 
I also removed additionalJOption, which I don't think that we are using.

Reply:
@jeenbroekstra do you know if removing the additionalJOption will cause any issues?